tests/telemetry/test_telemetry_metrics.py

```diff
@@ -427,11 +428,11 @@ def test_send_multiple_log_metric_no_duplicates_for_each_interval_check_time(tel
     with override_global_config(dict(_telemetry_dependency_collection=False)):
         for _ in range(3):
             sleep(0.1)
-            telemetry_writer.add_log("WARNING", "test error 1")
+            telemetry_writer.add_log(TELEMETRY_LOG_LEVEL.WARN, "test error 1")
```

Comment:
nit: We should use warning instead of warn. I think this is the convention used in pythons built in logging library. It would be nice to make it consistent.

Reply:
Fair point. I'll make that change. I thought it made sense to match what the backend expects but it makes more sense to match what's pythonic.